python/fichas/funcional.py

```diff
@@ -13,7 +13,7 @@
 
 
 def fact(n: int) -> int:
-    r"""
+    r""" #Found this 'r', is it necessary?
```

Comment:
Yes, the r stands for raw string. It lets you write special sequences of characters inside the docstring without escaping. In this case, I need `\times` and if I don't use the `r`, it will assume `\t` is a tab.

Read more about raw strings [here](https://www.askpython.com/python/string/python-raw-strings).